### ISSUE TITLE
Deploy online?

### ISSUE THREAD

**Issue by aviaryan** [anshumanv/ongaku-react#18]
Is this app live yet? I would love to try it out online. 
I see the `public/` folder so I guess the builds are working. We can have the builds in `gh-pages` branch to access it online. 

#### This issue is about suggesting an enhancement

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This issue is for enhancement/bug fixing.
- [ ] This issue is an addition to JSON database.
- [x] I have gone through [Contribution Guidelines](https://github.com/anshumanv/ongaku-react/blob/master/CONTRIBUTING.md).
- [ ] Assign this issue to me.

**Comment by filoxo** [comment]
As far as I know, its not yet been built anywhere to be tested. It's a great idea to put it up on a Github page in order to preview it, but I recently took a look at the original [ongaku](https://github.com/anshumanv/ongaku) repo and it looks pretty different from what I built a while ago. I haven't made the effort of reaching out to @anshumanv to talk about what needs to be updated or what direction things should go. 

By way of information, the `public/` dir is for static assets (as scaffolded by create-react-app) so its not an indication of anything really. CRA would build to a `build/` folder.

**Comment by aviaryan** [comment]
> By way of information, the public/ dir is for static assets (as scaffolded by create-react-app) so its not an indication of anything really. CRA would build to a build/ folder.

Oh, right! I totally confused `build` directory with `public`. 
Please let us know on this issue when the project is live.

**Comment by anshumanv** [comment]
Yes, consequently while @filoxo was building this (during hacktoberfest) the original HTML repo had a lot of contributors and UI was drastically changed. So all those changes need to be incorporated here as well post which we can have travis run the automated deployment process for us on gh-pages branch. Thanks for reaching out for this @aviaryan. I'll try to get things in place asap.

**Comment by anshumanv** [comment]
> I haven't made the effort of reaching out to @anshumanv to talk about what needs to be updated or what direction things should go.

It's cool, no worries, I'll update it to get the current view/UI aligned with the original repo quick and get this in production.
